### PR TITLE
Allows importing jira issues as link-relates

### PR DESCRIPTION
**Pull request by hegerj**

### CODE REVIEW

**Review by lukaszachy — CHANGES_REQUESTED** [review]
As discussed - usage of constants preferred.
Also please rebase to the current main (will simplify merging later depending on timing of other PRs )

**Comment by hegerj** [comment]
Updated the commit, also I think I am already based on latest main, aren't I?

File: tmt/convert.py [hunk 1]
```diff
@@ -713,7 +722,7 @@ def read_nitrate_case(testcase, makefile_data=None):
     except (KeyError, TypeError):
         pass
     for bug in testcase.bugs:
-        add_bug(bug.bug, data)
+        add_bug(bug.bug, data, bug.system)
```
Comment (thrix) [inline tmt/convert.py:725]: @hegerj does this mean tcms testcase will contain magic variables `1` or `2` for system? I would rather see some strings maybe?

Comment (psss) [reply tmt/convert.py:725]: This is how `nitrate` currently implements bugzilla/jira links, from the API you `1` for bugzilla bugs and `2` for jira issues.

**Review by lukaszachy — APPROVED** [review]

**Review by lukaszachy — APPROVED** [review]

**Pull request merged**